chatkit-core/src/main/kotlin/com/pusher/chatkit/users/api/UserSubscriptionDispatcher.kt

```diff
@@ -10,20 +11,25 @@ import com.pusher.chatkit.state.RoomUpdated
 import org.reduxkotlin.Dispatcher
 
 internal class UserSubscriptionDispatcher(
+    private val userApiTypeMapper: UserApiTypeMapper,
     private val joinedRoomsStateDiffer: JoinedRoomsStateDiffer,
     private val joinedRoomApiTypeMapper: JoinedRoomApiTypeMapper,
     private val dispatcher: Dispatcher
 ) {
 
     internal fun onEvent(event: UserSubscriptionEvent) {
         when (event) {
-            is UserSubscriptionEvent.InitialState ->
+            is UserSubscriptionEvent.InitialState -> {
+
+                dispatcher(CurrentUserReceived(
+                    userApiTypeMapper.toUserInternalType(event.currentUser)))
```

Comment:
Minor style: I think it's easier to follow if the number of brackets closing on each line matches the number that open
on the corresponding line, so I'd move the final two closes to a new line for balance:

```
                dispatcher(CurrentUserReceived(
                    userApiTypeMapper.toUserInternalType(event.currentUser)
                ))
```

Reply:
👍 yeah that makes sense!